modules/migration-adding-replication-repository-to-cam.adoc

```diff
@@ -28,6 +28,7 @@ You can add an object storage bucket as a replication repository to the CAM web
 ** *S3 provider access key*: Specify the `<AWS_SECRET_ACCESS_KEY>` for AWS or the S3 provider access key for MCG.
 ** *S3 provider secret access key*: Specify the `<AWS_ACCESS_KEY_ID>` for AWS or the S3 provider secret access key for MCG.
 ** *Require SSL verification*: Clear this check box if you are using a generic S3 provider.
+** If you use a custom CA bundle, click *Browse* and browse to the CA bundle file.
```

Comment:
Could explain the usage of button `Clear` somewhere?

Reply:
`Clear` is not a button. `Clear` is the term specified by the IBM Style Guide, p. 208, for unchecking a check box. Example: "Clear the **Password protected** check box."

Reply:
Could let customer to know the content of CA bundle file will be displayed in base64 encoded format?

Reply:
I'm not sure that's necessary because the customer does not need to view the file itself, but I have no objections to adding the format in parentheses.

Reply:
Doc updated